angular2-jwt.ts

```diff
@@ -1,6 +1,6 @@
 import {Injectable, Injector} from 'angular2/core';
 import {Http, HTTP_PROVIDERS, Headers, BaseRequestOptions, Request, RequestOptions, RequestOptionsArgs, RequestMethod, Response} from 'angular2/http';
-import {Observable} from 'rxjs/Observable';
+import {Observable} from 'rxjs';
```

Comment:
I thought the using `import {X} from 'y'` would not import everything, like `import * as rx from 'rxjs'`, or `import 'rxjs` do, but my understanding over imports/modules isn't great.

I'll back this line out...